lib/modules/manager/npm/post-update/yarn.ts

```diff
@@ -248,7 +248,9 @@ export async function generateLockFile(
         } else if (isYarnDedupeAvailable && s === 'highest') {
           commands.push(`yarn dedupe --strategy ${s}${cmdOptions}`);
         } else {
-          logger.debug(`yarn dedupe ${s} not available`);
+          commands.push(`npx yarn-berry-deduplicate --strategy ${s}`);
```

Comment:
Why not the `yarn dlx` approach?

Reply:
yes ! I just copied/pasted without thinking the line above
```suggestion
          commands.push(`yarn dlx yarn-berry-deduplicate --strategy ${s}`);
```